js/src/views/Signer/reducers/requests.js

```diff
@@ -108,3 +108,8 @@ function setIsSending (pending, id, isSending) {
     return p;
   }).slice();
 }
+
+function addTimestamp (request) {
+  request.date = new Date();
```

Comment:
The thing is that this new format would imply many changes for Components using theses requests. Maybe add this `timestamp` field at a lower level, like `request.payload.timestamp` ?